service/service.go

```diff
@@ -265,9 +283,12 @@ func New(config Config) (*Service, error) {
 // Boot starts top level service implementation.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		go s.AWSClusterController.Boot(context.Background())
-		go s.AzureClusterController.Boot(context.Background())
-		go s.KVMClusterController.Boot(context.Background())
+		prometheus.MustRegister(s.metricsCollector)
```

Comment:
The collector should be booted, which does the registration. You can see this in the other operators. Also see https://github.com/giantswarm/exporterkit/blob/master/collector/set.go.

Reply:
Yes sorry I missed some of the wiring. Getting it fixed.